Components/weatherData.type.ts

```diff
@@ -0,0 +1,14 @@
+export interface WeatherDataUsed {
+  forecasts: ForecastData[];
+  city: string;
+}
+export interface ForecastData {
+  id: number;
+  tempMin: number;
+  tempMax: number;
+  humidity: number;
+  rainPourcent: number;
```

Comment:
Actually, if it's already a percentage, you shouldn't have to multiply it by 100 later...
It's more `rainProbability` 

(btw, it's percent, not pourcent)